raspbian/applications/tvservice.md

```diff
@@ -64,4 +64,10 @@ When used in combination with the `--modes` options, displays the mode informati
 
 Extracts the display name from the EDID data and shows it.
 
+#### -l, --list
 
+Lists all attached displays and their display ID.
+
+#### -v, --device
```

Comment:
Should be 
```--device=display```
to match all the other commands that take parameters.

Otherwise fine by me.

Reply:
Fixed.